### PR TITLE
[TRIGGER] Calendly - New Event Created #6575

### PR DESCRIPTION
**Pull request by lcaresia**
### <samp>🤖 Generated by Copilot at 6cd06c8</samp>

This pull request adds four new components for different APIs and a new source component for Calendly events. The new components allow users to access and integrate data from Constant Contact, Google Palm API, Replicate, and Sellercloud in their Pipedream workflows. The new source component emits events when a new event is scheduled on Calendly.

### <samp>🤖 Generated by Copilot at 6cd06c8</samp>

> _This pull request has some new features_
> _For Calendly and other web creatures_
> _It adds four components_
> _For different endpoints_
> _And updates the `calendly_v2` package meters_

### <samp>🤖 Generated by Copilot at 6cd06c8</samp>

*  Bump up the version of the `@pipedream/calendly_v2` package to reflect the new source component for Calendly events ([link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-c7dd591fdb1e98e149c4091794eaa639cb25ce09b9cf4efd77cde191180e0c68L3-R3))
* Add the `calendly_v2-new-event-scheduled` source component that emits an event whenever a new event is scheduled on Calendly, using the Calendly API and a polling timer ([link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-545960752e770be196f03ae55d6a6f658ae3fc0f2fecd3f5eba0a040bd5059f8R1-R69))
* Deduplicate events based on their URI and store the last emitted event URI in the database ([link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-545960752e770be196f03ae55d6a6f658ae3fc0f2fecd3f5eba0a040bd5059f8R1-R69))
* Update the `pnpm-lock.yaml` file to track the dependencies and versions of the new or updated components for Constant Contact, Google Palm API, Replicate, and SellerCloud ([link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR866-R868), [link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR1765-R1767), [link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR3581-R3583), [link](https://github.com/PipedreamHQ/pipedream/pull/6581/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR3819-R3821))

### CODE REVIEW

File: components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import app from "../../calendly_v2.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "calendly_v2-new-event-scheduled",
+  name: "New Event Scheduled",
+  description: "Emit new event when a event is scheduled.",
```
Comment (dylburger) [inline components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]:
```suggestion
  description: "Emit a new event when an event is scheduled.",
```

Comment (dylburger) [reply components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]: Changed "Emit new event" to "Emit a new event" to follow Google's style guide on using articles.

Comment (dylburger) [inline components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]:
```suggestion
  description: "Emit a new event when an event is scheduled.",
```

Comment (dylburger) [reply components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]: Changed "Emit new event" to "Emit a new event" (Google's Developer docs style guide - Use articles)

Comment (dylburger) [inline components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]:
```suggestion
  description: "Emit a new event when an event is scheduled.",
```

File: components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
```
[the remark this answers is not in the document]
Comment (dylburger) [reply components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs:7]: Changed "Emit new event" to "Emit a new event" to follow Google's style guide on using articles.

**Comment by vunguyenhung** [comment]
Hi, I can see initial historical events and new events. Ready for release!
image

**Pull request merged**